src/template-linter.ts

```diff
@@ -221,17 +221,27 @@ export default class TemplateLinter {
   }
   async getFindUp(): Promise<FindUp> {
     if (!this._findUp) {
-      const { findUp } = await eval(`import('find-up')`);
+      const { findUp } = await import('find-up');
 
-      this._findUp = findUp;
+      this._findUp = findUp as FindUp;
     }
 
     return this._findUp;
   }
   private async templateLintConfig(cwd: string): Promise<string | undefined> {
     const findUp = await this.getFindUp();
 
-    return findUp('.template-lintrc.js', { cwd, type: 'file' });
+    const candidates = ['.template-lintrc.js', '.template-lintrc.cjs', '.template-lintrc.mjs'];
+
+    const results = await Promise.all(
```

Comment:
wondering if we could use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/race here

Reply:
I suppose it depends on what findUp returns when a file isn't found?

Reply:
ah, yep, it's not fail if not found